chunky/src/java/se/llbit/chunky/block/legacy/blocks/LegacyFlowerPot.java

```diff
@@ -83,4 +83,9 @@ public Tag getNewTagWithBlockEntity(Tag blockTag, CompoundTag entityTag) {
 
     return tag; // keep empty
   }
+
+  @Override
+  public boolean useBiomeTint() {
+    return true; // (in reality it is only used for fern)
```

Comment:
It should be fixed then.

Reply:
Returning true only for fern would be better. But from the function just above it seem the entityTag is needed to determine whether this is a fern or not. And because returning true when it should be false doesn't lead to a big adverse effect (it may lead to unnecessary work as more biome blending than necessary is performed), I just was lazy and made it simpler

Reply:
We could rename it to `canHaveBiomeTint` and merge this.

Reply:
Actually, this shouldn't be a problem at all. We replace all legacy blocks while loading a legacy chunk and let it create a new tag that takes the entity tag's information into account.

All the biome stuff is handled after converting the legacy blocks.